src/main/java/executor/command/CommandConvert.java

```diff
@@ -70,8 +72,16 @@ private Double extractAmount(CommandType commandType, String userInput) {
      * @return this function returns the 3 character unique string representing the currency to convert from
      */
     private String getCurrencyCovertFrom(String userInput) {
-        String fromStr = Parser.parseForFlag("from", userInput);
-        return fromStr;
+        try {
+            String fromStr = Parser.parseForFlag("from", userInput);
```

Comment:
again this doesn't really cause an error so it shouldn't be in the try-catch

Reply:
Yes i have removed this try catch which was unnecessary ;) Thanks